data-access-layer/bosh/BoshDirectorClient.js

```diff
@@ -771,6 +771,41 @@ class BoshDirectorClient extends HttpClient {
       );
   }
 
```

Comment:
just a question , we dont wait for the task to complete ? Or it will be part of the script which invoke these functions ?

Reply:
Yes--it would be the job of the client that calls these functions